team.go

```diff
@@ -557,7 +563,7 @@ func (client *Client) DeleteTeam(id ID) error {
 	return HandleErrors(err, m.Payload.Errors)
 }
 
-func (client *Client) RemoveMembers(team *TeamId, emails []string) ([]User, error) {
+func (client *Client) RemoveMembers(team *TeamId, memberships ...TeamMembershipUserInput) ([]User, error) {
```

Comment:
Should this be renamed to `RemoveMemberships`?

Reply:
I considered this and... yes sort of. I'm going to rename this method and create a new Deprecated method named `RemoveMembers`. That should ease the transition for anybody out there that hypothetically relies on this. Also this will necessitate updates in terraform-provider 👌